app/src/main/java/com/xmartlabs/template/ui/StartActivity.kt

```diff
@@ -15,16 +15,17 @@ class StartActivity : BaseAppCompatActivity() {
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
 
-    val session = sessionRepository.session
-    if (session == null) {
-      // TODO: handle on start activity no session
+//    val session = sessionRepository.session
```

Comment:
why? I think you'll need this. I agree that maybe not now, but you should mark this with a TODO or something as we usually don't leave any commented code